### ISSUE TITLE
Headings are Incorrectly Identified When Reviewing a contenteditable Node with the Virtual Cursor

### ISSUE THREAD

**Issue by jscholes** [FreedomScientific/VFO-standards-support#442]
#### Summary

When headings are included inside a node with `contenteditable="true"`, JAWS doesn't accurately convey the heading semantics and levels when reading the editable text with the Virtual Cursor (with Forms Mode deactivated in other words).  As an example of reproducing this problem:

1. Go to [https://jscholes.github.io/contenteditable-with-headings.html](https://jscholes.github.io/contenteditable-with-headings.html)
2. Using the JAWS Virtual Cursor, navigate through the editable div inside the main landmark with the Up and Down Arrow keys.

The editable div contains three headings.  The first and second are marked up as standard `<h1>` and `<h2>` elements, respectively, and the third is a `<div>` with `role="heading"` and `aria-level="3"`.

#### Expected result

JAWS will convey the role of "heading" for all of "Document Title", "Overview", and "Notes", as well as the correct heading level in each case.

#### Actual result

* In Firefox, JAWS fails to convey the "heading" role in all three cases.
* In Chrome, the "heading" role is announced but the level is always identified as "1" despite the mark-up of the second two headings being "2" and "3", respectively.

#### Example

[https://jscholes.github.io/contenteditable-with-headings.html](https://jscholes.github.io/contenteditable-with-headings.html)

##### JAWS version and build number

2020.2008.24 ILM

##### Operating System and version

Windows 10 Professional version 1909, build 18363.836

##### Browser and version:

* Mozilla Firefox 80.0.1 (64-bit)
* Google Chrome 85.0.4183.83 (Official Build) (64-bit)

CC @sinabahram

**Comment by JAWS-test** [comment]
I am rather surprised that headings in Chrome are recognized at all, then other elements are not recognized, e.g.
* lists (output without role in Firefox, no output in Chrome at all)
* tables (output without role in Chrome and Firefox)

The headings are also only output in Chrome when navigating within the area recognized as input field. However, the headings are not accessible via quick navigation (H) or overview (INS+F6)

I suspect the reason is that JAWS considers the area as an input field, and input fields can contain nothing but plain text

**Comment by jscholes** [comment]
> I suspect the reason is that JAWS considers the area as an input field, and input fields can contain nothing but plain text.

I suspect the same, but with `contenteditable` this is extremely problematic.

**Comment by JAWS-test** [comment]
> I suspect the reason is that JAWS considers the area as an input field, and input fields can contain nothing but plain text.

I am not sure if it is treated like an input field. When I use `role=textbox aria-multiline=true` instead of contenteditable, the output in Chrome is different (the headings are not recognized as such and the list entries are output, although without role)

**Comment by frex65** [comment]
Bug confirmed using JAWS 2020.2008.24 and Chrome 85.0.4183.121, where all headings are described as `<h1>`. In FF81 and IE11 no heading levels are output at all.
